include/in_process_server.h

```diff
@@ -235,6 +235,8 @@ class Client
     wl_surface* touched_window() const;
     std::pair<wl_fixed_t, wl_fixed_t> pointer_position() const;
     std::pair<wl_fixed_t, wl_fixed_t> touch_position() const;
+    bool pointer_events_clean() const; ///< If we have gotten a frame event since the last pointer event
```

Comment:
The name `pointer_events_clean()` doesn't say much to me (compared to, say, `!pointer_events_pending()`).

But is this even used?